### PR TITLE
Adding French Canadian language

### PR DESCRIPTION
**Pull request by DragonsLover**
Adding fr_CA language

### CODE REVIEW

File: src/main/resources/assets/endertech/lang/fr_CA.lang [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Note: these are GB spellings by default
```
Comment (Drayshak) [inline src/main/resources/assets/endertech/lang/fr_CA.lang]: Remove this comment and I think it's good to merge.

Comment (DragonsLover) [inline src/main/resources/assets/endertech/lang/fr_CA.lang]: Is it okay now?

**Comment by Drayshak** [comment]
Thanks!

**Comment by DragonsLover** [comment]
You're welcome! :)

**Pull request merged**